crypto/hpke/hpke.c

```diff
@@ -875,7 +875,6 @@ OSSL_HPKE_CTX *OSSL_HPKE_CTX_new(int mode, OSSL_HPKE_SUITE suite,
 void OSSL_HPKE_CTX_free(OSSL_HPKE_CTX *ctx)
 {
     if (ctx == NULL) {
```

Comment:
remove {}

Reply:
argh, yeah. I removed the ``ERR_raise`` call as it was more distracting than useful in the face of a test error.. But forgot the braces;-(